Cargo.toml

```diff
@@ -16,6 +16,7 @@ tari_utilities = { git = "https://github.com/tari-project/tari_utilities.git", t
 base64 = "0.10.1"
 blake2 = "0.9.1"
 bulletproofs = { package = "tari_bulletproofs", git = "https://github.com/tari-project/bulletproofs", tag = "v4.2.0" }
+bulletproofs_plus = { path = "../bulletproofs-plus", package = "tari_bulletproofs_plus"}
```

Comment:
You'll need to point this to github I think

Reply:
Yeah, looking at that now, forgot about it